Assets/Scripts/Framework/TargetSelector.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Framework
+{
+    public class TargetSelector : MonoBehaviour
```

Comment:
```suggestion
    public sealed class RouteHighlighter: MonoBehaviour
```

1. Added `sealed`
2. Vind de naam `TargetSelector` niet passen bij een route highlighter.